packages/plugins/src/plugins/eigenlayer/fetchers/deposit.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  ethereumNativeAddress,
+  ethereumNetwork,
+  NetworkId,
+  PortfolioAsset,
+  PortfolioElement,
+  PortfolioElementType,
+} from '@sonarwatch/portfolio-core';
+import { getEvmClient } from '../../../utils/clients';
+import { Cache } from '../../../Cache';
+
+import { eigenPodManagerAddress, platformId } from '../constants';
+import { getAddress } from 'viem';
+import { abi } from '../abi';
+import tokenPriceToAssetToken from '../../../utils/misc/tokenPriceToAssetToken';
+import BigNumber from 'bignumber.js';
+
+/**
+ * Returns the deposit positions for a given owner
+ * @param owner - The address of the owner
+ * @param cache - The cache instance
+ * @returns The deposit positions
+ */
+export const getDepositPositions = async (owner: string, cache: Cache) => {
+  const client = getEvmClient(NetworkId.ethereum);
+  const podManagerAddress = getAddress(eigenPodManagerAddress);
+
+  const podUserAddress = await client.readContract({
+    address: podManagerAddress,
+    abi: [abi.ownerToPod],
+    functionName: abi.ownerToPod.name,
+    args: [getAddress(owner)],
+  });
+
+  // If the user has no pod, the contract returns the ethereum native address address
+  if (!podUserAddress || podUserAddress === ethereumNativeAddress) return [];
+
+  // Get the balance of the pod user contract
+  const balance = await client.getBalance({
+    address: podUserAddress,
+  });
+
+  const assets: PortfolioAsset[] = [];
+  const tokenPrice = await cache.getTokenPrice(
+    ethereumNativeAddress,
+    NetworkId.ethereum
+  );
+  const asset = tokenPriceToAssetToken(
+    ethereumNativeAddress,
+    BigNumber(balance.toString())
+      .div(10 ** ethereumNetwork.native.decimals)
+      .toNumber(),
+    NetworkId.ethereum,
+    tokenPrice
+  );
+  assets.push(asset);
+
+  const elements: PortfolioElement = {
```

Comment:
💅 **[Nitpick]** – This is a minor stylistic suggestion and doesn't affect functionality. Feel free to ignore if you prefer.  

> ✏️ **Suggestion:** use singular instead of plural

```suggestion
  const element: PortfolioElement = {
```

Reply:
I also noticed the same in other modules.